app/services/CustomsOfficesService.scala

```diff
@@ -75,4 +80,13 @@ class CustomsOfficesService @Inject() (
   private def sort(customsOffices: Seq[CustomsOffice]): CustomsOfficeList =
     CustomsOfficeList(customsOffices.sortBy(_.name.toLowerCase))
 
+  def getCustomsOffices(implicit request: DataRequest[_]): CustomsOfficeList = {
```

Comment:
Purely semantic but I’d be tempted to pass userAnswers into this method rather than an implicit request. Also makes it easier to test